auth-web/src/router.ts

```diff
@@ -218,4 +255,21 @@ router.beforeEach((to, from, next) => {
   }
 })
 
+// Feature Flagging Routes
+// For Development only
+router.beforeEach((to, from, next) => {
+  if (to.matched.some(record => record.meta.conditionalRoute)) {
+    const flagCondition = LaunchDarklyService.getFlag('auth-web-dev-only')
```

Comment:
just a suggestion..plz discard if it doesn't make sense... we generally tend to name the LD variables using the feature itself..can auth-web-dev-only be renamed ? something like 'newbusinessincorporationfeature' or something more business/feature oriented? 
auth-web-dev-only is very generic and can be misused.. another dev might look at this and he might just add it to his feature as well which in turn can cause problems...

Reply:
For sure, great point. Initially i was thinking something generic would be a good idea so we could potentially re-use it in the future, but i can see both sides to that! 
Nitheesh has access to the LD web app, so i'll see if we can get this name updated :) Thanks!

Reply:
I think the LD admin website should already allow us to specify flags per environment... if this can be confirmed then we don't need "dev" in a flag name :)

Reply:
Yes, good point! We have updated the name to `incorporations-launch-feature`.